pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
we need to be able to set an entrypoint.  We need to be able to default the entry point to "bash -c" so the SCL logic "just works", and we need to allow users to choose a different entrypoint.

I also don't see why we wouldn't want to allow users to provide the WorkingDir as the k8s container api does, though it's not strictly necessary.